controllers/defaults.go

```diff
@@ -118,11 +117,24 @@ var (
 	// name of the lvm-operator container
 	LVMOperatorContainerName = "manager"
 
-	// default labels and values
-	DefaultLabelKey           = "app.lvm.openshift.io"
+	// labels and values
+
+	// AppKubernetesPartOfLabel is the Kubernetes recommended part-of label
+	AppKubernetesPartOfLabel = "app.kubernetes.io/part-of"
+	// AppKubernetesNameLabel is the Kubernetes recommended name label
+	AppKubernetesNameLabel = "app.kubernetes.io/name"
+	// AppKubernetesManagedByLabel is the Kubernetes recommended managed-by label
+	AppKubernetesManagedByLabel = "app.kubernetes.io/managed-by"
+	// AppKubernetesComponentLabel is the Kubernetes recommended component label
+	AppKubernetesComponentLabel = "app.kubernetes.io/component"
+
+	DefaultLabelKey           = "app.lvm.openshift.io/name"
 	TopolvmControllerLabelVal = "topolvm-controller"
 	TopolvmNodeLabelVal       = "topolvm-node"
 	VGManagerLabelVal         = "vg-manager"
+	ManagedByLabelValue       = "lvm-operator"
```

Comment:
some variables end with `val` and some with `value`.  We should make it consistent.